### PR TITLE
[ADD] New modules product_category_code and product_category_sequence

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

File: product_category_code/views/product_category_view.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="product_categ_form_view">
```
Comment (laetitia-gangloff) [inline product_category_code/views/product_category_view.xml]: set model after id :
<record id="product_categ_form_view" model="ir.ui.view">

File: product_category_sequence/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+from . import models
+from .models.product_category import update_null_and_slash_codes
+from . import tests
```
Comment (laetitia-gangloff) [inline product_category_sequence/__init__.py]: I am not sure it is correct to import tests here

**Comment by lmignon** [comment]
@laetitia-gangloff done

**Comment by laetitia-gangloff** [comment]
:+1:

**Pull request closed without merging**